README.md

```diff
@@ -19,43 +19,15 @@ The main concerns of a bare metal system are the inherent slowness in provisioni
 This project proposes a system that includes all of the above advantages and also addresses the fast provisioning issue for a bare metal system. Using this system, we propose to provision and release hundreds of nodes as quickly as possible with little impact on application performance.
 Current BMI (IMS) Architecture
 
-![](https://github.com/CCI-MOC/ims/blob/master/Selection_003.png)
+![](https://github.com/CCI-MOC/ims/blob/dev/Selection_003.png)
 
```

Comment:
Well, the thing is that updated image is in the dev branch at the moment. So, it will look fine now, but when we make this the master we would change the link at that time.